src/parsers/EN/ENRelativeDateFormatParser.js

```diff
@@ -0,0 +1,118 @@
+/*
+
+
+*/
+
+var moment = require('moment');
+var Parser = require('../parser').Parser;
+var ParsedResult = require('../../result').ParsedResult;
+var util  = require('../../utils/EN');
+
+var PATTERN = new RegExp('(\\W|^)' +
+    '(next|last|past)\\s*' +
+    '('+ util.INTEGER_WORDS_PATTERN + '|[0-9]+|few|half(?:\\s*an?)?)?\\s*' +
+    '(seconds?|min(?:ute)?s?|hours?|days?|weeks?|months?|years?)(?=\\s*)' +
+    '(?=\\W|$)', 'i'
+);
+
+var STRICT_PATTERN = new RegExp('(\\W|^)' +
+    '(next|last|past)\\s*' +
+    '('+ util.INTEGER_WORDS_PATTERN + '|[0-9]+)\\s*' +
+    '(seconds?|minutes?|hours?|days?)\\s*' +
+    '(?=\\W|$)', 'i'
+);
+
+exports.Parser = function ENRelativeDateFormatParser(){
+    Parser.apply(this, arguments);
+
+    this.pattern = function() {
+        return this.isStrictMode()? STRICT_PATTERN : PATTERN;
+    };
+
+    this.extract = function(text, ref, match, opt){
+
+        var index = match.index + match[1].length;
+        var modifier = match[2].toLowerCase().match(/^next/) ? 1 : -1;
```

Comment:
Instead of using group number directly, could you define the constant first? It's easier to debug.
```
var MODIFIER_WORD_GROUP = 2;
var MULTIPLYER_WORD_GROUP = 3;
...
```